sale_website_cleanup/static/description/index.html

```diff
@@ -0,0 +1,51 @@
+<head>
+  <style>
+      .backgrounds{background-color:#fff;color:#a41d35}
+  </style>
+</head>
+<section class="oe_container">
+ <div class="oe_row oe_spaced">
+  <p class="oe_mt32">
+   <strong>
+    This Addons add a cronjob that will canceled the orders that have created more than 24 hours.
```

Comment:
+This module add a cronjob that will cancel the orders that have been created
 +more than 24 hours.